src/app/course/course-question-snippet/course-question-snippet.component.html

```diff
@@ -142,7 +142,7 @@ <h2 class="tui-island__title">{{ uqj.question.title }}</h2>
                             </a>
                         </ng-template>
                         <a
-                            *ngIf="event.has_edit_permission"
+                            *ngIf="event.has_edit_permission && event.is_open"
```

Comment:
I don't think this is correct. The instructors cannot edit questions before or after the exam starts ??? It doesn't make sense to me.